spec/presenters/taxon_presenter_spec.rb

```diff
@@ -2,15 +2,20 @@
 
 RSpec.describe TaxonPresenter do
   describe "#payload" do
-    it "generates a valid payload" do
-      presenter = TaxonPresenter.new(
+    let(:presenter) do
+      TaxonPresenter.new(
         title: "My Title",
         base_path: "/taxons/my-taxon"
       )
+    end
+    subject { presenter.payload }
```

Comment:
When I read subject I thought of the class `TaxonPresenter` not the payload, I think it would be better to explicitly say payload, if this had been a larger file I would be mislead.